### DISCUSSION TITLE
Is support for PerformanceLog still needed in otobo.psgi ?

### DISCUSSION THREAD

**Discussion by bschmalhofer** [RotherOSS/otobo#1119] [Ideas]
In InterfaceAgent.pm there is code dedicated for performance logging. IMHO a better place for that would be in an Plack Middleware.

**Comment by stefanhaerter** [comment]
Is there a summary or documentation available what this does and what are the advantages and disadvantages?

**Comment by bschmalhofer** [comment]
No, it is more of a gut feeling that things like that should be implemented as middlewares instead of in the application code. "Separation of concerns" would be the keyword.
